MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -199,7 +199,9 @@ public ServerConfiguration()
 
     public bool EnableFolderView { get; set; } = false;
 
-    public bool EnableGroupingIntoCollections { get; set; } = false;
+    public bool EnableGroupingMoviesIntoCollections { get; set; } = false;
```

Comment:
Is there some sort of migration in place to update EnableGroupingIntoCollections into EnableGroupingMoviesIntoCollections ? (Assuming these are saved in the db)

Reply:
Happy to implement some migration logic if you point me to a good place for it. I am not very familiar with the project and you are looking at my first lines of C#.

Reply:
This PR just caught my interest so I am not 100% sure how the migration all hook together, but they are located here https://github.com/jellyfin/jellyfin/tree/master/Jellyfin.Server/Migrations

If this is indeed required someone from core can probably point you to a more specific spot.

Reply:
I may have figured out a nice way to migrate the configuration.